### PR TITLE
Preprod

### PR DESCRIPTION
**Pull request by mudsill**

### CODE REVIEW

File: content/admin/content/search/es.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and for version 2.x and version 5.x (using the `elasticSearch2` connector).
 
 Linkurious ships with an embedded {{es}} server (**version {{es.version}}**).
 
+> ATTENTION: The internal Elasticsearch is NOT suitable for large graph databases (>1,000,000 nodes). It cannot scale horizontally and is meant only to serve as a quick indexing strategy for POCs. We advise all production projects with ES--regardless of scale--to use an external installation.
```
[the remark this answers is not in the document]
Comment (davidrapin) [reply content/admin/content/search/es.md]: @mudsill Please word things in a less frightening way:
- the Embedded Elasticsearch's scalability is limited, it won't scale beyond 50'000'000 nodes
- it's intended for small-scale deployments or proofs or concept.
- the indexation and search performances depend on the RAM allocated to Elasticsearch, as well as the number and size of indexed properties

[the remark this answers is not in the document]
Comment (francescoinfante) [reply content/admin/content/search/es.md]: Used it up to 50M without issues on my laptop. @mudsill I would go with up to 50m

[the remark this answers is not in the document]
Comment (mudsill) [reply content/admin/content/search/es.md]: @davidrapin @francescoinfante updated

**Comment by davidrapin** [comment]
@mudsill thank you, merging

**Pull request merged**